operations/CSM_product_management/Post_Install_Customizations.md

```diff
@@ -767,6 +768,114 @@ Trial and error may be needed to determine what is best for a given system at sc
 
 * [Adjust HM Collector Ingress Replicas and Resource Limits](../hmcollector/adjust_hmcollector_resource_limits_requests.md)
 
+### `cray-cfs-api` pods are `OOMKilled`
+
+Increase the memory requests and limits associated with `cray-cfs-api` deployment in the `services` namespace.
+
+1. (`ncn#`) Get the current cached customizations.
```

Comment:
Should this have the user change into some temp directory? The files this guide generates are never deleted, so they'll linger around the system.

Reply:
These steps were taken from a section earlier on.  We've discovered another problem with the whole procedure in that it overwrites the stored manifest for the system, so this is going to need some work in general.